data_types/softlayer_load_balancer.go

```diff
@@ -0,0 +1,25 @@
+package data_types
+
+type SoftLayer_Load_Balancer_Update_Parameters struct {
+	Parameters []SoftLayer_Load_Balancer_Update `json:"parameters"`
+}
+
+type SoftLayer_Load_Balancer_Update struct {
+	SecurityCertificateId *int `json:"securityCertificateId"`
+}
+
+type SoftLayer_Load_Balancer struct {
```

Comment:
From a comparison of fields, I believe this corresponds to the `SoftLayer_Network_Application_Delivery_Controller_LoadBalancer_VirtualIpAddress` service.  I am wondering if straying this far from the documented SoftLayer API nomenclature won't cause more confusion, particularly among developers who are already familiar with the SoftLayer API.

Example scenario: I am working on another datatype with a relational property of type SoftLayer_Network_Application_Delivery_Controller_LoadBalancer_VirtualIpAddress.  My own inclination would be to do a quick search of the softlayer-go datatypes directory, and, not finding a match on the required name, proceed to create my own (duplicate) version.